app/javascript/packs/form-steps-wait.tsx

```diff
@@ -118,7 +120,7 @@ export class FormStepsWait {
       body: new window.FormData(form),
     });
 
-    if ('polyfill' in window.fetch) {
+    if ((window.fetch as FetchOrFetchPolyfill).polyfill) {
```

Comment:
Change from `in` to truthy check, since `in` will pass for the stubbed value of `undefined`.

The polyfill sets it to `true` ([source](https://github.com/github/fetch/blob/fb5b0cf42b470faf8c5448ab461d561f34380a30/fetch.js#L604)).

Reply:
Yeah, the only way it could work previously is if we would delete the property altogether (`delete fetch.polyfill`), but there's not a clean with to stub that.